src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext, useEffect } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleList } from '../components/PeopleList';
+import { PeopleContext } from '../store/PeopleContext';
+
+export const PeoplePage: React.FC = () => {
+  const { people, loading, errorMessage, loadPeople } =
+    useContext(PeopleContext);
+
+  useEffect(() => {
+    loadPeople();
```

Comment:
Ensure that `loadPeople` is a stable function reference to avoid unnecessary re-renders. If `loadPeople` is defined within a component and changes on every render, consider using `useCallback` to memoize it.